Scripts/SAPSetupAssistantValidationScripts/SetupAssistantSAPValidateNetworkConnection.ps1

```diff
@@ -0,0 +1,29 @@
+param (
+   [string]$hostName,
```

Comment:
While `hostName` is okay for a variable name, PowerShell conventions typically use Pascal Case for parameter names.

Reply:
You can also make these arguments required by decorating them with the following.

```
[Parameter(Mandatory=$true)]
```

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.4#parameter-attribute

Reply:
I have to update canvas app if I update the variable names, I am going to leave them as is for now. But will keep in mind for future scripts